blog/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+# Create your models here.
+class Post(models.Model):
+    
+    store_name =  models.CharField(max_length=100)
+    reportcontent = models.TextField(default = '')
+    location = models.TextField(default = '')
+    image = models.ImageField(blank = True, upload_to='blog')
```

Comment:
```suggestion
    image = models.ImageField(blank=True, upload_to='blog')
```

인자로 값을 넣을때는 연산자 기준으로 띄어쓰기 안하셔도 됩니다~

Reply:
넵 수정했습니다!